### ISSUE TITLE
aria-current="false" is read as aria-current="true"

### ISSUE THREAD

**Issue by ShadowBB** [nvaccess/nvda#7892]
### Steps to reproduce:

1. Make HTML file with the following txt:
[ARIA current.txt](https://github.com/nvaccess/nvda/files/1603092/ARIA.current.txt)

2. Open the page (tested with Firefox, Chrome and IE)
3. Navigate to the the third list item ("item false").
4. Notice the list item being announced as "Current".

### Expected behavior:
The third list item is not announced as "current"

### Actual behavior:
The third list item is announced as "current"

### System configuration:
NVDA version:
2017.4

NVDA Installed or portable:
Installed

Other information:
NA

Windows version:
Windows 10 Home Version 1709 Build 16299.125

### Other questions:

Does the issue still occur after restarting your PC?
Yes.

Have you tried any other versions of NVDA?
No.

**Comment by bramd** [comment]
I think this might be according to the ARIA spec (see https://www.w3.org/TR/wai-aria-1.1/#aria-current):

> The aria-current attribute is an enumerated type. Any value not included in the list of allowed values SHOULD be treated by assistive technologies as if the value true had been provided.

False is an allowed value, but it is the default and indicated by omitting a value or omitting the whole aria-current attribute. Some feedback of the spec authors would be welcome here.

**Comment by leonardder** [comment]
JAWS 2018 treats a value of false as real false.

@bramd: Note that, even though false is the default, it is an allowed 
value in the enumeration as you pointed out. Only non-allowed values 
should be treated as true.

**Comment by bramd** [comment]
That's right. The only point of confusion here might be that there are 
more attributes which are true is any value is set, but when I'm just 
reading this part of the spec I think false should be allowed as a value 
that is equal to false.

**Comment by Adriani90** [comment]
closing as works for me, solved in #7901. I have tested in IE11, Chrome 71 and Firefox 64.0.2. I cannot reproduce it. @ShadowBB please comment on this if it is still occuring and we will reopen it. Thanks for reporting.
